etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,38 @@
+
+# Firejail profile for Parsec
+# Description: Remote desktop application focused on gaming and other 3D applications
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include parsecd.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.parsec
+whitelist ${HOME}/.parsec
+whitelist /usr/share/parsec
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+
+# Due to the nature of parsec, the following directives will not work:
+# - no3d
+# - novideo
+# - noaudio
+# - noinput (it does remote passthrough stuff for gamepads)
+# - private-dev (because of the above)
+apparmor
+nodvd
+nonewprivs
+nogroups
+notv
+nou2f
+noroot
+
+# Will not start with zenity missing
+private-bin parsecd,zenity
+private-tmp
+
+dbus-user none
+dbus-system none
+
+memory-deny-write-execute
```

Comment:
Is it a GTK program?

Reply:
Unsure, quite probably not. It does some custom things afaik.